### PR TITLE
Constant input attr added to fully connected operation in TFLite frontend

### PR DESCRIPTION
**Pull request by d-smirnov**
This PR adds an ability to handle constant input attr to "fully connected" operation. Unit tests amended.

### CODE REVIEW

File: tests/python/frontend/tflite/test_forward.py [hunk 1]
```diff
@@ -2456,25 +2456,27 @@ def test_forward_sparse_to_dense():
...
 
+    # Initializes the input tensor with array containing incrementing
+    # numbers from 1.
+    data_array = np.arange(1, total_size_1 + 1, dtype=np.float32)
+    filter_array = np.arange(1, total_size_2 + 1, dtype=np.float32)
+
     with tf.Graph().as_default():
-        in_data = array_ops.placeholder(shape=tensor_in_sizes, dtype='float32')
-        in_filter = constant_op.constant(filter_array, shape=filter_in_sizes, dtype='float32')
+        in_name="input"
```
Comment (mbaret) [inline tests/python/frontend/tflite/test_forward.py:2474]:
```suggestion
        in_name = "input"
```

File: python/tvm/relay/frontend/tflite.py [hunk 2]
```diff
@@ -1695,10 +1695,9 @@ def convert_fully_connected(self, op):
             raise ImportError("The tflite package must be installed")
 
         input_tensors = self.get_input_tensors(op)
-        assert len(input_tensors) >= 2, "input tensors length should be >= 2"
+        assert len(input_tensors) in (2, 3), "input tensors length should be two or three"
```
Comment (mbaret) [inline python/tvm/relay/frontend/tflite.py:1698]: input_tensors can now have 3 elements, but I can't see the 3rd tensors (input_tensors[2]) being used anywhere?

File: python/tvm/relay/frontend/tflite.py [hunk 3]
```diff
@@ -1720,7 +1719,7 @@ def convert_fully_connected(self, op):
         # Dense expected Weight shape: [out_dim, n_units]
         # Dense output shape: [batch_size, out_dim]
         target_shape = tuple((-1, weight_tensor_shape[1]))
-        in_expr = self.get_expr(input_tensor_idx)
+        in_expr = self.get_tensor_expr(input_tensor)
```
Comment (mbaret) [inline python/tvm/relay/frontend/tflite.py:1722]: Is this a related change?

File: python/tvm/relay/frontend/tflite.py (hunk 2 above)
Comment (d-smirnov) [reply python/tvm/relay/frontend/tflite.py:1698]: line 1756. Related to bias, if it is provided.

File: python/tvm/relay/frontend/tflite.py (hunk 3 above)
Comment (d-smirnov) [reply python/tvm/relay/frontend/tflite.py:1722]: Yes

File: tests/python/frontend/tflite/test_forward.py (hunk 1 above)
Comment (d-smirnov) [reply tests/python/frontend/tflite/test_forward.py:2474]: Some networks uses literal value instead of tensor placeholder for first argument. wrap flag switches the wrapping of the actual value of first operand.

File: python/tvm/relay/frontend/tflite.py (hunk 2 above)
Comment (mbaret) [reply python/tvm/relay/frontend/tflite.py:1698]: So is this a fix, or is it related to adding support for constant input?

Comment (d-smirnov) [reply python/tvm/relay/frontend/tflite.py:1698]: Providing that assertion was made stricter, this is a small improvement.

**Comment by mbaret** [comment]
I'm still not really sure what's meant by the 'wrap' and can't find anything in the tensorflow/tflite docs named that. It appears to just control whether the input is a constant, in which case could we make it something more explicit like 'const_input'?

**Comment by d-smirnov** [comment]
@siju-samuel, @anijain2305  please take a look

File: tests/python/frontend/tflite/test_forward.py [hunk 4]
```diff
@@ -2555,25 +2555,27 @@ def test_forward_sparse_to_dense():
...
-    total_size_2 = 1
-    for s in tensor_in_sizes:
-        total_size_1 *= s
-    for s in filter_in_sizes:
-        total_size_2 *= s
-    # Initializes the input tensor with array containing incrementing
-    # numbers from 1.
-    data_array = [f * 1.0 for f in range(1, total_size_1 + 1)]
-    filter_array = [f * 1.0 for f in range(1, total_size_2 + 1)]
+    total_size_1 = np.prod( tensor_in_sizes )
```
Comment (anijain2305) [inline tests/python/frontend/tflite/test_forward.py:2561]: Minor - please remove the space between before and after `filter_in_sizes`

Comment (d-smirnov) [reply tests/python/frontend/tflite/test_forward.py]: Done

**Comment by anijain2305** [comment]
Thanks @d-smirnov @mbaret This is merged!
